core/src/epicli/data/common/ansible/playbooks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+  - hosts: all
```

Comment:
I think this can be removed and you may need to add `logging` role to `hosts` in core\src\epicli\data\common\ansible\playbooks\filebeat.yml (line 4):
```
- hosts: elasticsearch:kibana # to gather facts
  tasks: []
```